sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md

```diff
@@ -0,0 +1,18 @@
+| 403    | <ul><li>For Substatus 3 (Write Forbidden): </br><ul><li>This exception occurs when a geo-replicated database account runs into writable/readable location changes (say, after a failover).</li><li>This exception can occur regardless of the Consistency level set for the account. </li><li>The client refreshes it's locational endpoints and retries requests when the user has enabled endpoint discovery in their client (default behavior).</li></ul></li><li>For all other cases: </br><ul><li> The client does NOT retry requests when this exception is encountered. </li>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| 404/1002    | <ul><li>For write operations: </br><ul><li>If multiple write locations are enabled for the account, the SDK will fetch the write endpoints and retry once per each of these. </li><li>The client refreshes it's locational endpoints and retries requests when the user has enabled endpoint discovery in their client (default behavior).</li><li>If the account does not have multiple write locations enabled, the SDK will retry only once. </li></ul></li><li>For read operations: </br><ul><li>If multiple write locations are enabled for the account, the SDK will fetch the read endpoints and retry once per each of these. </li><li>The client refreshes it's locational endpoints and retries requests when the user has enabled endpoint discovery in their client (default behavior).</li><li>If the account does not have multiple write locations enabled, the SDK will retry only once. </li>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
I think in java and .net SDK, for 410/1002, we will first retry up to 5s, something to maybe change in Python, 1 time may not be enough

Reply:
</ul></li><li>For read operations: </br><ul><li>If multiple `write locations` are enabled  ->  should it be `read locations`

Reply:
The policy looks like this for the additional logic described with refreshing the endpoints:
![image](https://user-images.githubusercontent.com/30335873/225687718-ea76cc50-74fe-43ab-81ef-7454f894323b.png)
Otherwise it will only do one retry, instead of one retry per read endpoint available.